schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
```

Comment:
### Optional
- [ ] It would be better if we make the id field not null autoincrement and also add a primary key to the `id` field. Also, going forward let us try to capitalize our commit messages